cpp/velox/operators/writer/VeloxParquetDatasource.cc

```diff
@@ -45,9 +45,7 @@ const int32_t kGzipWindowBits4k = 12;
 
 void VeloxParquetDatasource::init(const std::unordered_map<std::string, std::string>& sparkConfs) {
   if (strncmp(filePath_.c_str(), "file:", 5) == 0) {
-    auto path = filePath_.substr(5);
-    auto localWriteFile = std::make_unique<LocalWriteFile>(path, true, false);
-    sink_ = std::make_unique<WriteFileSink>(std::move(localWriteFile), path);
+    sink_ = dwio::common::FileSink::create(filePath_, {.pool = pool_.get()});
```

Comment:
`pool` should not be needed for `localFileSink` and `hdfsFileSink`.  You can take a check at velox layer.

Reply:
Correct me if I'm wrong. I refer [velox code](https://github.com/facebookincubator/velox/blob/c5bcc33e93678e5bf0a7479ba16212035008a302/velox/dwio/common/tests/LocalFileSinkTest.cpp#L41-L50) to initialize the sink, could you please share velox layer?

Reply:
Yes. Seems `pool` is always provided in velox's usage even it's not actually used.